doc/manual/parallel-computing.rst

```diff
@@ -463,8 +463,8 @@ variable takes on all values added to the channel. An empty, closed channel
 causes the ``for`` loop to terminate.
 
 
-Shared Arrays (Experimental)
------------------------------------------------
+Shared Arrays
+-------------
```

Comment:
Ironically, `DArray`s proved to be the shorter-lived object, so I think we can safely remove the "Experimental" tag.